src/main/resources/config/config-template.conf

```diff
@@ -25,5 +25,12 @@ generation: {
     amountOfGridGenerators: Int | 10 # Amount of actors actually building the grids
     amountOfRegionCoordinators: Int | 5 # Amount of actors coordinating generation per municipality
     distinctHouseConnections: Boolean | false # If there shall be distinct lines for house connection
+    osm: {
+      filter: { # filter to be applied for the LvOsmoGridModel
+        building: [string]
```

Comment:
Are we able to provide default values for the filter strings? Or can we make them even optional?

Reply:
Made the filter optional in config and renamed `PbfFilter` to `SourceFilter`. Furthermore, I implemented filter instantiation in `LvCoordinator`.